pact/yaml/testnet/relay.yaml

```diff
@@ -0,0 +1,23 @@
+codeFile: ../../relay/relay.pact
+signers:
+  - public: ef711514b5a248ac16ffd5cc94511dfccbd89005e9737343dfbb3743c8a63ca6
+data:
+  ns: "relay"
+  upgrade: false
+  relay-coin-account: relay-bank
+  lockup: 365
+  bond: 100000
+  activity: 365
+  endorsers: 10
+  denouncers: 15
+  confirm: 0.5
+  rate: 0.000822
+  fee: 1.0
```

Comment:
Just to confirm, that's KDA100k, for one year, 10 endorsers, 15 denouncers, 1.0KDA fee. LGTM 👍

Reply:
Yep. These have changed since, too, and are still only final for testnet.